### PR TITLE
Header with nav component

### PR DESCRIPTION
**Pull request by dominiquecuevas**
Features fixed navigation bar at top with links to 
- Home
- Order Eggs
- Gallery
- Recipes
- FAQ
- About the Farm
- Login

Order Eggs page is a dummy account to demonstrate linking. Other non-Home pages expected to return 404 errors

https://github.com/athelia/full-spectrum-frontend/assets/56096225/ae2bd11d-8e25-42fd-93b1-a0f769479a41

Screen Shot 2023-06-23 at 4 28 15 PM

### CODE REVIEW

**Comment by athelia** [comment]
Add summary of changes please!

File: components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const Header: React.FC = () => {
+  return (
+    <nav className="navbar">
+      <Link href="/">Home</Link>
+      <Link href="/order-eggs">Order Eggs</Link>
+      <Link href="/gallery">Gallery</Link>
+      <Link href="/recipes">Recipes</Link>
+      <Link href="/faq">FAQ</Link>
+      <Link href="/about">About the Farm</Link>
+      <Link href="/about">Login</Link>
```
Comment (athelia) [inline components/Header.tsx]: typo `/about`

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,6 +1,13 @@
 import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
+import Layout from '@/components/Layout'
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return (
+    <>
```
Comment (athelia) [inline pages/_app.tsx]: nit: extra brackets

File: pages/_app.tsx [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
+      <Layout>
+        <Component {...pageProps} />
+      </Layout>
+    </>
```
Comment (athelia) [inline pages/_app.tsx]: nit: extra brackets

File: components/Layout.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from "react"
+import Header from "./Header"
+
+const Layout: React.FC<{children: ReactNode}> = (props) => {
+  return (
+    <>
```
Comment (athelia) [inline components/Layout.tsx:6]: nit: delete extraneous brackets

File: components/Layout.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+      <Header />
+      <div className="main">
+        {props.children}
+      </div>
+    </>
```
Comment (athelia) [inline components/Layout.tsx:11]: nit: extra brackets

File: pages/order-eggs.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+const OrderEggs = () => {
+  return (
+    <>
```
Comment (athelia) [inline pages/order-eggs.tsx:3]: nit: extra brackets

File: pages/order-eggs.tsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+      <h1>Order Eggs</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+    </>
```
Comment (athelia) [inline pages/order-eggs.tsx:8]: nit: extra brackets

**Review by athelia — CHANGES_REQUESTED** [review]
- one issue on Header with wrong link for login
- lots of nits of extra brackets
- missing tests

File: pages/order-eggs.tsx (hunk 6 above)
Comment (dominiquecuevas) [reply pages/order-eggs.tsx:3]: just kidding, we need the outside brackets here to wrap the multiple child elements

File: components/Layout.tsx (hunk 4 above)
Comment (dominiquecuevas) [reply components/Layout.tsx:6]: just kidding, we needed these outside brackets to wrap around multiple elements

File: pages/order-eggs.tsx (hunk 7 above)
Comment (dominiquecuevas) [reply pages/order-eggs.tsx:8]: we need the brackets here to wrap around multiple components

**Review by athelia — APPROVED** [review]

File: pages/order-eggs.tsx (hunk 6 above)
Comment (athelia) [reply pages/order-eggs.tsx:3]: use `<React.fragment>`

**Pull request merged**